### ISSUE TITLE
Message Additions

### ISSUE THREAD

**Issue by rodrigok** [RocketChat/Rocket.Chat.Apps-engine#55]
@graywolf336 commented on [Fri May 04 2018](https://github.com/RocketChat/Rocket.Chat.Apps-ts-definition/issues/44)

Allow Apps to provide message attachments/uploads.

![image](https://user-images.githubusercontent.com/850391/39647715-ab28d996-4fa5-11e8-9a54-c021ee369670.png)

**Comment by matheusbsilva137** [comment]
Hi, @rodrigok!

Message attachments (such as audio, images, videos and documents) are already available in our latest version of the Apps-Engine and these can be inserted within a `MessageBuilder` object or using the `attachments` attribute from the `Message` class.
May this issue be closed?

**Comment by graywolf336** [comment]
It can not be closed. This issue is about adding support for new items to the `plus` icon in the message box inside rocket.chat

**Comment by matheusbsilva137** [comment]
Ok, @graywolf336. Should this issue be transferred to the [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) repository, then?

**Comment by graywolf336** [comment]
No. It is a feature we have wanted since the first days of the Apps (Rocketlets) conception. An app should be able to add a new entry in the "Add files from". For example, Google Drive, FTP, SFTP, Dropbox, etc.
